myscript.js

```diff
@@ -0,0 +1,19 @@
+function validate(){
```

Comment:
Why did you name this function validate? A better straight forward name would be naming it what it validates i.e `validateAge`

Reply:
Changed it